src/histolab/scorer.py

```diff
@@ -67,16 +67,20 @@ def __call__(self, tile: Tile) -> float:
 
 
 class NucleiScorer(Scorer):
-    r"""Implement a Scorer that estimates the presence of nuclei in an H&E-stained tile.
+    r"""This class implements an hybrid algorithm that combines thresholding and
```

Comment:
The first line of the docstring should be ONE line and self-contained, then space then an explanation.. so I would keep the old first line and then put the explanation